virl/files/mitaka.vinstall.py

```diff
@@ -453,7 +453,7 @@ def building_salt_all():
 keystone.password: {ospassword}
 keystone.tenant: admin
 keystone.tenant_id: {tenid}
-keystone.auth_url: {auth_url}/
+keystone.auth_url: 'http://127.0.0.1:5000/v2.0/'
```

Comment:
be more specific...on the mitaka image thats built the images are being installed so what is being broken